tests/ingest/test_models.py

```diff
@@ -1,11 +1,10 @@
-from mex.backend.ingest.models import _BaseBulkIngestRequest
```

Comment:
you can also remove this file entirely.

the test was only needed to test the `_BaseBulkIngestRequest.get_all` method. but since that is gone, `mex.backend.ingest.models` does not contain any actual code anymore. it's just definitions of models